etna/models/nn/deepstate/deepstate.py

```diff
@@ -333,6 +347,98 @@ def __init__(
             split_params=split_params,
         )
 
+    def _validate_timestamp(self, ts: TSDataset):
```

Comment:
Do we really need this?

Reply:
That was our specification. Are you against some specific checks here or against this whole function?

Reply:
> Are you against some specific checks here or against this whole function?

I don't understand why we should create special checks here. It's additional code that is not important imho


It just inflates codebase and make future refactoring more complex without any profit  just more intutive error